content/blog/peace-of-mind-with-cloud-secret-providers/index.md

```diff
@@ -0,0 +1,854 @@
+---
+title: "Peace of Mind with Cloud Secret Providers"
+
+date: 2020-05-07T14:30:35-07:00
+draft: false
+meta_desc: "Encrypt your Pulumi secrets with a cloud secrets provider"
+meta_image: meta.png
+authors:
+    - lee-briggs
+tags:
+    - features
+    - security
+---
+
+The secrets in your infrastructure are a vital part of your security model, and provisioning infrastructure is an inherently privileged process. [Previously](https://www.pulumi.com/blog/managing-secrets-with-pulumi/) we introduced secret encryption and started encrypting secret configuration values inside the Pulumi state so that users could be confident their passwords, tokens and other secret values were viewable only by them while managing their infrastructure.
```

Comment:
```suggestion
The secrets in your infrastructure are a vital part of your security model, and provisioning infrastructure is an inherently privileged process. [Previously]({{< relref "/blog/managing-secrets-with-pulumi" >}}) we introduced secret encryption and started encrypting secret configuration values inside the Pulumi state so that users could be confident their passwords, tokens and other secret values were viewable only by them while managing their infrastructure.
```